src/index.js

```diff
@@ -1,13 +1,39 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const tasks = [
+  {
+    description: 'Take out the trash',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Wash the dishes',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'Buy groceries',
+    completed: false,
+    index: 3,
+  },
+];
 
```

Comment:
- [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider `Array.sort((a, b) => a.index - b.index)`